File: alura/carreira_java/example35/src/br/com/flaviogf/Book.java
package br.com.flaviogf;

public class Book {
    private String title;
    private String author;
    private String description;

    public Book(String title, String author, String description) {
        this.title = title;
        this.author = author;
        this.description = description;
    }

    @Override
    public String toString() {
        if (description.length() <= 30) {
            return String.format("Title: %s\nAuthor: %s\nDescription:%s", title, author, description);
        }

        return String.format("Title: %s\nAuthor: %s\nDescription:%s", title, author, description.substring(0, 27) + "...");
    }
}

File: pluralsight/buildinganasyncapiwithaspnetcore/src/Books.Api/Entities/Book.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Books.Api.Entities
{
    [Table("Books")]
    public class Book
    {
        [Key]
        public Guid Id { get; set; }

        [Required]
        [MaxLength(150)]
        public string Title { get; set; }

        public string Description { get; set; }

        [MaxLength(2500)]
        public Guid AuthorId { get; set; }

        public Author Author { get; set; }
    }
}
